nibabel/tests/test_tripwire.py

```diff
@@ -16,9 +16,6 @@ def test_tripwire():
     with pytest.raises(TripWireError):
         silly_module_name.do_silly_thing
     # Check AttributeError can be checked too
-    try:
+    with pytest.raises(AttributeError) as err:
         silly_module_name.__wrapped__
-    except TripWireError as err:
-        assert isinstance(err, AttributeError)
-    else:
-        raise RuntimeError('No error raised, but expected')
+    assert isinstance(err.value, AttributeError)
```

Comment:
This test was basically verifying that `TripWireError` subclassed `AttributeError`. If we want it to do something different than the `with:` block:

```suggestion
    assert isinstance(err.value, TripWireError)
```

Otherwise I think it would be okay to drop:

```suggestion
```